README.rst

```diff
@@ -94,6 +94,12 @@ Notable Features
 - Seamless integration with `webhooks <https://github.com/python-telegram-bot/python-telegram-bot/wiki/Webhooks>`_ and `polling <https://docs.python-telegram-bot.org/en/stable/telegram.ext.application.html#telegram.ext.Application.run_polling>`_
 - `Comprehensive documentation and examples <#working-with-ptb>`_
 
+Free threading
```

Comment:
can this be moved to the Concurrency section below? We have some comments on threading there already and it feels weird splitting the comments up